dor/providers/parsers.py

```diff
@@ -123,10 +128,17 @@ def get_struct_maps(self):
     def get_resource(self):
         return PackageResource(
             id=self.get_id(),
-            alternate_identifier=self.get_alternate_identifier(),
+            alternate_identifier= self.get_alternate_identifier(),
             type=self.get_type(),
             events=self.get_preservation_events(),
             metadata_files=self.get_metadata_files(),
             data_files=self.get_data_files(),
             struct_maps=self.get_struct_maps(),
         )
+
+    def _apply_relative_path(self, descriptor_path: Path, path_to_apply: str) -> str:
+        if path_to_apply.startswith("../"):
```

Comment:
I need to think some more about this; we don't want a path relative to the running directory (`cwd`) --- that's too much path. Working in the console doing the following gives the result I was expecting.

```
combined_path = (descriptor_path / path_to_apply)
new_relative_path = combined_path.relative_to(descriptor_path / "..")
```

Reply:
Yes, you are correct.I tried a different approach, but when I run the tests from the CWD, it causes the paths to be resolved from there. This creates issue because I need the paths to be relative to a specific directory, not the CWD.